### ISSUE TITLE
Add galaxy scripts

### ISSUE THREAD

**Issue by natefoo** [galaxyproject/gravity#7]
A lot of the things in Galaxy's `scripts/` directory probably could/should be moved to gravity. At least `helper.py`

**Comment by bgruening** [comment]
@natefoo are scripts that interact with the TS welcome here? Like the following:

 * `galaxy toolshed update`
 * `galaxy toolshed upgrade`
 * `galaxy toolshed dist-upgrade`

**Comment by natefoo** [comment]
Yes, absolutely. If you are developing please develop against the click branch.
